src/ResourceManager/TrafficManager/Commands.TrafficManager2/packages.config

```diff
@@ -1,4 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Azure.Management.TrafficManager" version="2.3.0-preview" targetFramework="net452" />
-</packages>
+  <package id="Microsoft.Azure.Management.TrafficManager" version="2.4.0-preview" targetFramework="net452" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.4.0" targetFramework="net452" />
```

Comment:
@tatlicioglu same comment about removing the references to `Microsoft.Rest.ClientRuntime`, `Microsoft.Rest.ClientRuntime.Azure` and `Newtonsoft.Json` from this file